quickwit/Cargo.toml

```diff
@@ -118,6 +118,7 @@ num_cpus = "1"
 numfmt = "1.1.1"
 once_cell = "1"
 oneshot = "0.1.5"
+opendal = { version ="0.44", default-features = false }
```

Comment:
can we feature gate this to make the development a tiny bit lighter?
(You can mimick what is done for azure.)